spec/serializers/user_serializer_spec.rb

```diff
@@ -196,4 +196,33 @@
       expect(json[:custom_fields]['secret_field']).to eq(nil)
     end
   end
+
+  context "with user_api_keys" do
+    let(:user) { Fabricate(:user) }
+
+    let(:user_api_key_0) { Fabricate(:readonly_user_api_key, user: user) }
+    let(:user_api_key_1) { Fabricate(:readonly_user_api_key, user: user) }
+    let(:user_api_key_2) { Fabricate(:readonly_user_api_key, user: user) }
+    let(:user_api_key_3) { Fabricate(:readonly_user_api_key, user: user) }
+    let(:user_api_key_4) { Fabricate(:readonly_user_api_key, user: user) }
+
+    before do
+      freeze_time
+
+      user_api_key_0.update_attributes(last_used_at: 2.days.ago, revoked_at: Time.now)
+      user_api_key_1.update_attributes(last_used_at: 7.days.ago)
+      user_api_key_2.update_attributes(last_used_at: 1.days.ago)
+      user_api_key_3.update_attributes(last_used_at: 4.days.ago, revoked_at: Time.now)
```

Comment:
We prefer `Time.zone.now` in our code base to avoid any discrepancy in the time zone being used.